tests/_test_mscolab/test_mscolab.py

```diff
@@ -114,3 +117,51 @@ def test_handle_db_seed(self):
         assert len(all_users) == 10
         all_permissions = Permission.query.all()
         assert len(all_permissions) == 17
+
+    def test_handle_mscolab_certificate_init(self):
+        handle_mscolab_certificate_init()
+        FILE_KEY = os.path.join(mscolab_settings.MSCOLAB_SSO_DIR, 'key_mscolab.key')
```

Comment:
In doubt you can trigger pylint, which gives also a good expression about "problems" 
here is an output of "one" configuration :) When we add a .pylintrc we have to discuss which options we want to ignore, beforehand.

```
pylint test_mscolab.py 
pylint: Command line or configuration file:1: UserWarning: Specifying exception names in the overgeneral-exceptions option without module name is deprecated and support for it will be removed in pylint 3.0. Use fully qualified name (maybe 'builtins.Exception' ?) instead.
************* Module tests._test_mscolab.test_mscolab
test_mscolab.py:43:0: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:52:0: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:68:0: C0115: Missing class docstring (missing-class-docstring)
test_mscolab.py:68:0: C0103: Class name "Test_Mscolab" doesn't conform to '[A-Z_][a-zA-Z0-9]+$' pattern (invalid-name)
test_mscolab.py:86:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:109:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:121:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:123:8: C0103: Variable name "FILE_KEY" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)
test_mscolab.py:125:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
test_mscolab.py:129:8: C0103: Variable name "FILE_CRT" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)
test_mscolab.py:131:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
test_mscolab.py:136:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:138:8: C0103: Variable name "FILE_KEY" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)
test_mscolab.py:140:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
test_mscolab.py:144:8: C0103: Variable name "FILE_CRT" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)
test_mscolab.py:146:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
test_mscolab.py:151:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:153:8: C0103: Variable name "FILE_YAML" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)
test_mscolab.py:155:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
test_mscolab.py:160:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:165:4: C0116: Missing function or method docstring (missing-function-docstring)
test_mscolab.py:30:0: C0411: standard import "import argparse" should be placed before "import pytest" (wrong-import-order)
```